README.md

```diff
@@ -0,0 +1,69 @@
+# Capstone Module 1
+
+Final Project Using HTML,CSS and JAVASCRIPT (Microverse Module 1 Curriculum)
+
+<p align="center">
+  <img src="./images/screenshot/mobile-version.png"/>
+</p>
+
+<hr>
+
+<p align="center">
+  <img src="./images/screenshot/desktop-version.png"/>
+</p>
+
+
+This project is meant to test student ability's on being able to put together all that they have learnt in module1 to ensure mastery in core concepts.
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+
+Here's a [video ]() of me giving a few description of the project.
+
+## Live Demo
+
+[Live Demo Link](https://kelomo2502.github.io/Capstone-Project1/)
+
```

Comment:
**[REQUIRED BUT OPTIONAL]**

- Though this was missed by the previous reviewer,  please kindly add the link to the presentation video to meet the project requirement before you merge.